src/mixins/atoms.js

```diff
@@ -63,52 +61,32 @@ export const AtomsMixin = (superclass) => class extends superclass {
         return sphereMesh;
     }
 
-    /**
-     * Convert THREE.vector to an XYZ coordinate array
-     * @param {THREE.Vector} vector THREE vector
-     * @return {Number[]}
-     * @private
-     */
-    _vectorToXYZCoordinate(vector) {return [vector.x, vector.y, vector.z]}
-
     _getDefaultSettingsForElement(element = this.settings.defaultElement, scale = this.settings.atomRadiiScale) {
         return {
             color: this.getAtomColorByElement(element),
             radius: this.getAtomRadiusByElement(element, scale),
         }
     }
 
-    /**
-     * Converts coordinates from the global (world) space to local wrt object3d
-     * @param {THREE.Object3D} object3d - The object to set new reference frame.
-     * @param {Number[]} worldCoordinate - Coordinate of the object in the Global (World) frame.
-     * @return {Number[]}
-     * @private
-     */
-    _convertXYZCoordinateToLocal(object3d, worldCoordinate) {
-        return this._vectorToXYZCoordinate(object3d.worldToLocal(new TV3(...worldCoordinate)));
-    }
-
-    drawAtomsAsSpheres(atomRadiiScale) {
-
-        const drawGroup = this.materialGroup;
-        this.getAtomColorByElement();
-
-        const basisWithRepetitions = Made.tools.basis.repeat(this.basis, Array(3).fill(this.settings.atomRepetitions));
-        const sphereMeshObjects = basisWithRepetitions.coordinates.map((atomicCoordinate, atomicIndex) => {
-            const element = basisWithRepetitions.getElementByIndex(atomicIndex);
-            // local coordinate with respect to the drawGroup
-            const coordinate = this._convertXYZCoordinateToLocal(drawGroup, atomicCoordinate.value);
+    createSitesGroup(basis, atomRadiiScale) {
+        const sitesGroup = new THREE.Group();
```

Comment:
Should this be BasisGroup? Why Sites?

Reply:
This is a group of SphereMesh objects each representing a site, hence sitesGroup.

Reply:
Yes, but what is a Site? Are you referring to https://study.com/academy/lesson/crystal-lattice-definition-structure.html? Then the word seems misused

Reply:
Renamed to `atomsGroup`.